internal/mode/static/nginx/config/servers_template.go

```diff
@@ -33,10 +33,6 @@ server {
 
         {{ range $l := $s.Locations }}
     location {{ $l.Path }} {
-        {{ if $l.Internal -}}
```

Comment:
The Internal boolean in the Location struct is no longer needed, so it can be removed in the Go code.

Reply:
updating